### DISCUSSION TITLE
Add cowboy hat and lever action rifle to game

### DISCUSSION THREAD

**Discussion by Spoon2Spoon** [FakeFishGames/Barotrauma#7908] [Ideas and suggestions]
Hey I have an idea that I'd really like in the game that I know many people would greatly enjoy, including myself and many of my friends who i got into barotrauma. In the lobby select for barotrauma the casual lobby setting has a man wearing a cowboy hat and holding a lever action rifle, and I think it's a really big let down that despite being able to use a revolver and the idea of Europa being a wild west you aren't able to actually be a cowboy. That's why I'd like to suggest the idea of adding a cowboy hat and lever gun to the game, along with possibly other cowboy stuff.

**Comment by Ek-Videogames** [comment]
**YEE** - and i cannot stress this enough - **Haw**

**Reply by cshingle** [reply]
Cowboys and Clowns should hate each other. More roleplay = better.
